core/cmd/local_client_test.go

```diff
@@ -183,14 +176,24 @@ ROOT: %[1]s
 SECURE_COOKIES: true
 SESSION_TIMEOUT: 15m0s
 TELEMETRY_INGRESS_LOGGING: false
-TELEMETRY_INGRESS_SERVER_PUB_KEY: 
-TELEMETRY_INGRESS_URL: 
-CHAINLINK_TLS_HOST: 
+TELEMETRY_INGRESS_SERVER_PUB_KEY:
+TELEMETRY_INGRESS_URL:
+CHAINLINK_TLS_HOST:
 CHAINLINK_TLS_PORT: 6689
 CHAINLINK_TLS_REDIRECT: false`, cfg.RootDir())
 
-	if !strings.Contains(msg, expected) {
-		t.Errorf("Expected to find:\n\n%s\n\nWithin:\n\n%s\n\nDiff:\n\n%s", expected, msg, diff.Diff(expected, msg))
+	logs, err := cltest.ReadLogs(lcfg.Dir)
+	require.NoError(t, err)
+
+	for _, envVarLog := range strings.Split(expected, "\n") {
+		msg := cltest.FindLogMessage(logs, func(msg string) bool {
+			return strings.HasSuffix(msg, envVarLog)
+		})
+		require.NotEmpty(t, msg, "No env var message found")
+
+		if !strings.Contains(msg, envVarLog) {
+			t.Errorf("Expected to find:\n\n%s\n\nWithin:\n\n%s\n\nDiff:\n\n%s", msg, envVarLog, diff.Diff(expected, msg))
+		}
```

Comment:
How did the format change here?

Reply:
We don't have JSON anymore... this was depending on the JSON format to extract the info and validate it. As we now log in console format, we need to go line by line to find matches.